### PR TITLE
Update Wrangler assets

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
This PR loads the Wrangler banner from the Wrangler repo, and replaces the gif with an embedded (and copy pastable) asciinema recording made for [this PR](https://github.com/cloudflare/wrangler/pull/868).

https://asciinema.org/a/SXJNxWUO8GF5T7N8R2mMMvE1W

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
       "integrity": "sha512-lZEV1ZANzfzSYBU6WHSErsy7jLPbD1iIgAboASPMcKo7woVni5/5IKWeT0RxC8rY802MFktur3OKEw2JY1Tv2w==",
       "dev": true,
       "requires": {
-        "@babel/plugin-proposal-async-generator-functions": "7.2.0",
-        "@babel/plugin-proposal-dynamic-import": "7.5.0",
```
Comment (victoriabernard92) [inline package-lock.json]: Is there a reason you prefer the package-lock to not lock to a specific version?

Comment (EverlastingBugstopper) [reply package-lock.json]: i didnt do this on purpose it's an artifact of me running `wrangler preview`.. not sure what the deal is i can revert these changes

Comment (signalnerve) [reply package-lock.json]: package-lock stuff is so weird 😤 do you mind removing these changes from this PR?

Comment (EverlastingBugstopper) [reply package-lock.json]: yes! done :)

**Pull request merged**